rapid_pro_tools/rapid_pro_client.py

```diff
@@ -272,6 +272,45 @@ def get_raw_messages(self, created_after_inclusive=None, created_before_exclusiv
 
         return raw_messages
 
+    def get_groups(self, uuid=None, name=None):
+        """
+        Gets all matching contact groups from a rapid_pro workspace
+
+        :param uuid: group UUID
+        :type uuid: str
+        :param name: group name
+        :type group name: str
+        :return: group query
+        :rtype: list of temba_client.v2.types.Group
+        """
+
+        return [group for group in self.rapid_pro.get_groups(uuid=uuid, name=name).all(retry_on_rate_exceed=True)]
+
+    def get_contacts(self, uuid=None, urn=None, group=None, deleted=None, before=None, after=None, reverse=None):
+        """
+        Gets all matching contacts from a rapid_pro workspace
+
+        :param uuid: contact UUID
+        :type uuid: str
+        :param urn: contact URN
+        :type urn: str
+        :param group: contact group name or UUID
+        :type group: str
+        :param deleted: return deleted contact only
+        :type deleted: bool
+        :param reverse: whether to return contacts ordered in reverse (oldest first).
+        :type reverse: bool
+        :param before: modified before
+        :type before: datetime
+        :param  after: modified after
+        :type before: datetime
```

Comment:
Elsewhere in this file we called this last_modified_before_exclusive and last_modified_after_inclusive. I suggest doing the same here so it's super unambiguous what the filter means without needing to check the api docs

Reply:
Updated thanks.